alembic/versions/8f086f8c0c21_move_weather_data_to_separate_table.py

```diff
@@ -0,0 +1,70 @@
+"""Move weather data to separate table
+
+Revision ID: 8f086f8c0c21
+Revises: 3e1852612f34
+Create Date: 2021-03-30 10:30:39.368099
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8f086f8c0c21'
+down_revision = '3e1852612f34'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'weathers',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('modified', sa.DateTime(), nullable=False),
+        sa.Column(
+            'weather_info', postgresql.JSONB(astext_type=sa.Text()), nullable=True
+        ),
+        sa.Column('retrieved_at', sa.DateTime(), nullable=True),
+        sa.Column('telescope_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ['telescope_id'], ['telescopes.id'], ondelete='CASCADE'
+        ),
+        sa.PrimaryKeyConstraint('id'),
+    )
+    op.create_index(
+        op.f('ix_weathers_created_at'), 'weathers', ['created_at'], unique=False
+    )
+    op.create_index(
+        op.f('ix_weathers_telescope_id'), 'weathers', ['telescope_id'], unique=True
+    )
+    op.drop_column('telescopes', 'weather')
+    op.drop_column('telescopes', 'weather_retrieved_at')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        'telescopes',
+        sa.Column(
+            'weather_retrieved_at',
+            postgresql.TIMESTAMP(),
```

Comment:
Should this be of type `sa.DateTime()`?

Reply:
This is what alembic specifies for a `sa.DateTime` column. And the migration checks seem to like this, so seems :+1: